src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -1202,6 +1202,11 @@ def _get_authorization_code_worker(authority_url, resource, results):
             break
         except socket.error as ex:
             logger.warning("Port '%s' is taken with error '%s'. Trying with the next one", port, ex)
+        except UnicodeDecodeError:
+            logger.warning("Please make sure there is no international (Unicode) character in the computer name "
```

Comment:
Just curious about that if we actually have international character in the compute name, the only way to get it passed is to rename the compute name ?

Reply:
The link does have instructions about how to bypass it: https://github.com/Azure/azure-cli/issues/12957

But using Unicode as a computer name is always not a good idea.

Reply:
Got it.